Sources/QueuesFluentDriver/JobModelMigrate.swift

```diff
@@ -2,23 +2,34 @@ import protocol SQLKit.SQLDatabase
 import enum SQLKit.SQLColumnConstraintAlgorithm
 import enum SQLKit.SQLDataType
 import enum SQLKit.SQLLiteral
+import struct SQLKit.SQLQualifiedTable
 import struct SQLKit.SQLRaw
 
 public struct JobModelMigration: AsyncSQLMigration {
+    private let jobsTableString: String
+    private let jobsTable: SQLQualifiedTable
+
     /// Public initializer.
-    public init() {}
-    
+    public init(
+        jobsTableName: String = "_jobs_meta",
+        jobsTableSpace: String? = nil
+    ) {
+        self.jobsTableString = "\(jobsTableSpace.map { "\($0)_" } ?? "")\(jobsTableName)"
+        self.jobsTable = .init(jobsTableName, space: jobsTableSpace)
+    }
+
     // See `AsyncSQLMigration.prepare(on:)`.
     public func prepare(on database: any SQLDatabase) async throws {
         let stateEnumType: String
         
         switch database.dialect.enumSyntax {
         case .typeName:
-            stateEnumType = "\(JobModel.schema)_storedjobstatus"
-            try await database.create(enum: stateEnumType)
-                .value("pending")
-                .value("processing")
-                .run()
+            stateEnumType = "\(self.jobsTableString)_storedjobstatus"
+            var builder = database.create(enum: stateEnumType)
+            for `case` in StoredJobState.allCases {
```

Comment:
Would it be cleaner to call this `state` instead of having to escape `case`? Or even do something like 

```swift
StoredJobState.allCases.forEach { builder.value($0.rawValue) }
```
to closer match what happens on L34?

Reply:
If you want to think that way, I should write it `builder = StoredJobState.reduce(builder, { $0.value($1.rawValue) })` to stay in keeping with my preference of always using the return value even though it's guaranteed to always be `self` (call it stylistic pedantry on my part 😅).

Not to say that's not a good idea, 'cause it is 😆